File: projects/m1/004-units-of-time/solutions/DomenicoTraina/units-of-time.py
 # this program will converte days,hours,minutes,seconds to only seconds.
days=int(input("How many days? "))
hours=int(input("How many hours? "))
minutes=int(input("How many minutes? "))
seconds=int(input("How many seconds? "))

secondbyminutes=minutes*60
secondbyhours=hours*3600
secondbydays=days*24*3600

totalsecond=seconds+secondbyminutes+secondbyhours+secondbydays

print("In {} days, {} hours, {} minutes, {} seconds, there are exactly {} seconds ".format(days,hours,minutes,seconds,totalsecond))

File: projects/m1/004-units-of-time/solutions/biagio/units-of-time.js
const prompt = require('prompt-sync')();

var days = prompt("How many days: ");
var hours = prompt("How many hours: ");
var minutes = prompt("How many minutes: ");
var seconds = prompt("How many seconds: ");

var daysToSeconds = days * 24 * 60 * 60;
var hoursToSeconds = hours * 60 * 60;
var minutesToSeconds = minutes * 60;

var totalSeconds = daysToSeconds + hoursToSeconds + minutesToSeconds + seconds;

console.log("Total seconds is: " + totalSeconds);